apps/web/src/components/professor-card.tsx

```diff
@@ -83,32 +87,63 @@ const SocialLinks = ({ professor }: ProfessorCardProps) => {
 
 export const ProfessorCard = ({ professor, ...props }: ProfessorCardProps) => {
   return (
-    <div {...props}>
-      <div className="border-blue-1000 bg-beige-300 flex h-full flex-col items-start gap-2.5 rounded-[1.25rem] border p-1">
-        <div className="flex w-[240px] grow flex-col">
-          <div className="border-blue-1000 flex flex-col rounded-t-2xl border bg-orange-500 px-5 py-2">
-            <span className="text-beige-300 break-words text-xl font-semibold">
+    <div
+      className="flex  flex-wrap lg:max-w-[280px] lg:max-h-[443px] p-2 md:p-7 gap-5 rounded-2xl border-t border-t-newGray-4 bg-newGray-6 shadow-course-navigation mt-8"
```

Comment:
Cards seem too rounded on mobile

We should increase the amount of cards displayed on high-low / medium screens, it feels empty on the side